capture/src/capture.cpp

```diff
@@ -41,6 +43,51 @@ void SigInt( int )
     s_disconnect.store(true, std::memory_order_relaxed);
 }
 
+static bool s_isStdoutATerminal = false;
+
+void InitIsStdoutATerminal() {
+#ifdef _WIN32
+    s_isStdoutATerminal = _isatty( fileno( stdout ) );
+#else
+    s_isStdoutATerminal = isatty( fileno( stdout ) );
+#endif
+}
+
+bool IsStdoutATerminal() { return s_isStdoutATerminal; }
+
+#define ANSI_RESET "\033[0m"
+#define ANSI_BOLD "\033[1m"
+#define ANSI_BLACK "\033[30m"
+#define ANSI_RED "\033[31m"
+#define ANSI_GREEN "\033[32m"
+#define ANSI_YELLOW "\033[33m"
+#define ANSI_MAGENTA "\033[35m"
+#define ANSI_CYAN "\033[36m"
+#define ANSI_ERASE_LINE "\033[2K"
+
+// Like printf, but if stdout is a terminal, prepends the output with
+// the given `ansiEscape` and appends ANSI_RESET.
+void AnsiPrintf( const char* ansiEscape, const char* format, ... ) {
+  if( IsStdoutATerminal() )
+  {
+    // Prepend ansiEscape and append ANSI_RESET.
+    char buf[256];
+    va_list args;
+    va_start( args, format );
+    vsnprintf( buf, sizeof buf, format, args );
+    va_end( args );
+    printf( "%s%s" ANSI_RESET, ansiEscape, buf );
+  }
+  else
+  {
+    // Just a normal printf.
+    va_list args;
+    va_start( args, format );
+    vfprintf( stdout, format, args );
+    va_end( args );
+  }
+}
```

Comment:
Please also use 4 spaces for indentation.

Reply:
done